op-deployer/docs/version-support.md

```diff
@@ -0,0 +1,13 @@
+## Version Support Policy
+
+Each version of OP Deployer supports:
+
+- The latest development version of the smart contracts at the time of the OP Deployer release
+- The most recent governance-approved release of the smart contracts at the time of the OP Deployer release
+
+The below table outlines which smart contract versions each OP Deployer version supports:
+
+| OP Deployer Version | Supported L1 Versions                             | Supported L2 Versions                     | 
+|---------------------|---------------------------------------------------|-------------------------------------------|
+| v0.0.1 - v0.0.10    | `op-contracts/v1.6.0`                             | `op-contracts/v1.7.0-beta.1+l2-contracts` |
+| v0.0.11             | `op-contracts/v1.6.0`, `op-contracts/v1.8.0-rc.4` | `op-contracts/v1.7.0-beta.1+l2-contracts` |
```

Comment:
Should this table to be expanded to include a column for which OPCM versions are compatible with each OP Deployer version? Or is that a completely unrelated vector?

Reply:
OPCM is deployed one-per-version, so it's unrelated to this table

Reply:
Sorry I'm having a hard time wrapping my head around all the versions. The pr description mentions:
```
I separated the deployment path into two branches for v1.6.0 and v1.7.0.
```
What versions are you referring to there? And when does opcm v170 get used versus opcm v200